src/components/Calls/CallNotification.js

```diff
@@ -1,30 +1,36 @@
 import { h } from 'preact';
 import { useState } from 'preact/compat';
 
+import { Livechat } from '../../api';
 import I18n from '../../i18n';
 import PhoneAccept from '../../icons/phone.svg';
 import PhoneDecline from '../../icons/phoneOff.svg';
 import constants from '../../lib/constants';
 import { Avatar } from '../Avatar';
 import { Button } from '../Button';
 import { createClassName, getAvatarUrl } from '../helpers';
+import { CallIframe } from './CallIFrame';
 import styles from './styles.scss';
 
 
-export const CallNotification = ({ callProvider, callerUsername, url, dispatch } = { callProvider: undefined, callerUsername: undefined, dispatch: undefined }) => {
+export const CallNotification = ({ callProvider, callerUsername, url, dispatch, rid } = { callProvider: undefined, callerUsername: undefined, dispatch: undefined, rid: undefined }) => {
 	const [show, setShow] = useState(!!callProvider && !!callerUsername && !!dispatch && !!url);
+	const [isIframe, setIframe] = useState(false);
```

Comment:
Again, not a good practise to store the state if the Iframe component within `CallNotification` component